### PR TITLE
Selfcites extension

### PR DESCRIPTION
**Pull request by kaplun**

### CODE REVIEW

**Comment by kaplun** [comment]
CC @tsgit This should solve performance issues (at least I don't see any). And fix the truncation of the authorlist.

**Comment by kaplun** [comment]
Note: untested.

**Comment by tsgit** [comment]
looks good. I'll test locally.

File: modules/bibrank/lib/bibrank_selfcites_indexer.py [hunk 1]
```diff
@@ -144,33 +136,32 @@ def compute_self_citations(recid, tags, authors_fun):
...
                 get_collaborations_from_record(cit, tags))
             if collaborations.intersection(cit_collaborations):
                 self_citations.add(cit)
-    else:
-        # Use authors names
-        for cit in citers:
-            cit_authors = get_authors_from_record(cit, tags)
-            if (not authors or len(cit_authors) > 20) and \
+                # It's a selfcite: no need for more info.
+                break
```
Comment (tsgit) [inline modules/bibrank/lib/bibrank_selfcites_indexer.py]: do you mean `break` or `continue` ?

**Comment by tsgit** [comment]
so does this actually still require the arbitrary `CFG_SELFCITES_AUTHOR_LIMIT` or can that be done away with? if there still is need for it, then `CFG_SELFCITES_AUTHOR_LIMIT` should go into the actual config file for this bibrank method
`....etc/bibrank/selfcites.cfg` or `invenio-local.cnf`

-- in principle -- but I won't worry about that for legacy.

File: modules/bibrank/lib/bibrank_selfcites_indexer.py (hunk 1 above)
Comment (kaplun) [reply modules/bibrank/lib/bibrank_selfcites_indexer.py]: right. I meant continue

**Comment by kaplun** [comment]
I think it's cleaner with your flag. I'll integrate it.

**Comment by tsgit** [comment]
@kaplun ping

**Pull request merged**